source/gear/swordofthesun-base.js

```diff
@@ -0,0 +1,44 @@
+const { GearTemplate } = require('../classes');
+const { ESSENCE_MATCH_STAGGER_FOE } = require('../constants');
+const { getModifierCategory } = require('../modifiers/_modifierDictionary');
+const { dealDamage, changeStagger, combineModifierReceipts, removeModifier, generateModifierResultLines } = require('../util/combatantUtil');
+const { damageScalingGenerator } = require('./shared/scalings');
+
+module.exports = new GearTemplate("Sword of the Sun",
```

Comment:
Don't forget to add the Sword of the Sun and Let Sleeping Dogs Lie to the Everything Bagel.